addons/web_tour/static/src/tour_service/tour_interactive.js

```diff
@@ -363,23 +388,22 @@ export class TourInteractive {
 
     _onMutation() {
         if (this.currentAction) {
-            let tempAnchor = hoot.queryFirst(this.currentAction.anchor, { visible: true });
+            const tempAnchors = this.findTriggers(this.currentAction.anchor);
+            const anchorEl = this.anchorEls[0];
+            let tempAnchor = tempAnchors[0];
             tempAnchor = tempAnchor && this.getAnchorEl(tempAnchor, this.currentAction.event);
-            if (
-                (!this.anchorEl && tempAnchor) ||
-                (this.anchorEl && tempAnchor && tempAnchor !== this.anchorEl)
-            ) {
-                this.anchorEl = tempAnchor;
+            if ((!anchorEl && tempAnchor) || (anchorEl && tempAnchor && tempAnchor !== anchorEl)) {
+                this.anchorEls = tempAnchors;
                 this.removeListeners();
                 this.setActionListeners();
-            } else if (!tempAnchor && this.anchorEl) {
+            } else if (!tempAnchor && anchorEl) {
                 this.pointer.hide();
-                this.anchorEl = tempAnchor;
+                this.anchorEls = tempAnchors;
                 if (!hoot.queryFirst(".o_home_menu", { visible: true })) {
                     this.backward();
                 }
+                this.updatePointer(this.anchorEls[0]);
```

Comment:
You should maybe use arrays and their length, instead of taking the first element and checking these ones only. 
The first element can change and should be check for the pointer, but only that.